05_stringer/nickolee/stringer.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+type IPAddr [4]byte
+
+func (ip IPAddr) String() string {
+	return fmt.Sprintf("%d.%d.%d.%d", ip[0], ip[1], ip[2], ip[3])
+}
+
+func main() {
+	fmt.Fprint(out, IPAddr{127, 0, 0, 1})
+
```

Comment:
Little picky one, but this extra new line is inconsistent with your other functions.

Reply:
Please fix.

No need for blank lines at beginning or end of block.
Use blank lines between type, method and function definitions and within a block for logical separation.

Reply:
Got it thanks for the tip! Noted and have fixed this.